library/modules/bigip_device_auth_radius.py

```diff
@@ -0,0 +1,754 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright: (c) 2018, F5 Networks Inc.
+# GNU General Public License v3.0 (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['stableinterface'],
+                    'supported_by': 'certified'}
+
+DOCUMENTATION = r'''
+---
+module: bigip_device_auth_radius
+short_description: Manage RADIUS authentication on a BIG-IP
+description:
+  - Manage the RADIUS authentication configuration.
+version_added: 2.9
+options:
+  servers:
+    description:
+      - Specifies RADIUS servers configurations for use with RADIUS authentication profiles
+    type: list
+    suboptions:
+      server:
+        description:
+          - The IP address of the server.
+          - This field is required.
+            In that case, the simple list can specify server IPs. See examples for
+            more clarification.
+        type: str
+      port:
+        description:
+          - The port of the server.
+          - If no port number is specified, the default port C(1812) is used.
+        type: int
+        default: 1812
+      secret:
+        description:
+          - Specifies secret used for accessing RADIUS server.
+        type: str
+      timeout:
+        description:
+          - Specifies the timeout value in seconds.
+          - The default timeout value is C(3) seconds.
+        type: int
+        default: 3
```

Comment:
why is there a default? ansible modules do not set defaults on items sent to the device